frontend/e-learning-fe/src/components/choices.js

```diff
@@ -0,0 +1,15 @@
+function Choices({ choice }) {
```

Comment:
let's rename this to `Choice` since it is rendering a single choice not a list of choices

Reply:
Sir regarding this one po, sa SELS-12 lang po sya nagagamit since static lang po dun na idisplay lang pero sa other PRs ko po onwards kay I did not use the choices component na sir.

Reply:
we are in SELS-13 already and we are still using this, pa update na lang po if hindi na gagamitin :bow:

Reply:
Resolving this comment We will stop using this in SELS-14